ubercj/hacktoberfest-tracking
# Change document title and favicon

# Description #
#21 Change the document title to represent the page the user is one.  Example would be when on the login page it would be 'Login • Hacktoberfest Tracking'.  Also adding a new favicon to represent that his project is for Hacktoberfest.

- Create store to hold the state for the page title
- Added Hacktoberfest svg as favicon
- Setting component name to title store for Login, Dashboard, and Account (although may not be needed if it won't have it's own page)

# Testing #

1. Go to login page
2. Validate document title is 'Login • Hacktoberfest Tracking'
3. Login into app
4. Validate once Dashboard page appears that document title is 'Dashboard • Hacktoberfest Tracking'

# Screenshots #
Login Page
<img width="232" alt="Screen Shot 2022-10-13 at 7 35 17 PM" src="https://user-images.githubusercontent.com/14244920/195733729-477872f8-44a9-4617-aabd-f739986db091.png">

Dashboard Page
<img width="232" alt="Screen Shot 2022-10-13 at 7 36 16 PM" src="https://user-images.githubusercontent.com/14244920/195733758-b2207dfa-1c3a-4845-8d53-7c5b1e730678.png">
